controllers/disruption_controller.go

```diff
@@ -386,7 +386,7 @@ func (r *DisruptionReconciler) updateInjectionStatus(instance *chaosv1beta1.Disr
 
 	terminationStatus := disruptionTerminationStatus(*instance, chaosPods)
 	if terminationStatus != tsNotTerminated {
-		switch instance.Status.InjectionStatus {
+		switch status {
```

Comment:
is there a reason I'm not seeing for why we're not using the status variable?

Reply:
Instance status is updated by the method `disruptionTerminationStatus` hence the variable value won't be the same IMO